src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+                System.out.println("Got it. I've added this task:");
+                toBePrinted = "[" + entries.get(entriesCount).getSymbol() + "] ["
+                        + entries.get(entriesCount).getStatusIcon() + "] " + entries.get(entriesCount).description;
+                System.out.println("   " + toBePrinted);
+                entriesCount++;
+                System.out.println("Now you have " + entriesCount + " tasks in the list.");
+                try {
+                    writeToFile(dukeDataPath, toBePrinted + System.lineSeparator());
+                } catch (IOException e) {
+                    System.out.println("Something went wrong: " + e.getMessage());
+                }
+                break;
+            case "event":
+                int eventDateTimeIndex = userCommandDetails.indexOf("/at");
+                String eventDescription = userCommandDetails.substring(0, eventDateTimeIndex - 1);
+                String eventDateTime = userCommandDetails.substring(eventDateTimeIndex);
+                entries.add(entriesCount, new Event(eventDescription, eventDateTime));
+                System.out.println("Got it. I've added this task:");
+                toBePrinted = "[" + entries.get(entriesCount).getSymbol() + "] ["
+                        + entries.get(entriesCount).getStatusIcon() + "] " + entries.get(entriesCount).description;
+                System.out.println("   " + toBePrinted);
+                entriesCount++;
+                System.out.println("Now you have " + entriesCount + " tasks in the list.");
+                try {
+                    writeToFile(dukeDataPath, toBePrinted + System.lineSeparator());
+                } catch (IOException e) {
+                    System.out.println("Something went wrong: " + e.getMessage());
+                }
+                break;
+            case "done":
+                String stringTaskIndex = userMessage.substring(userMessage.indexOf(" ") + 1);
+                int intTaskIndex = Integer.parseInt(stringTaskIndex) - 1;
+                entries.get(intTaskIndex).isDone = true;
+                printLongLine();
+                System.out.println("Nice! I've marked this task as done:\n"
+                        + "   [" + entries.get(intTaskIndex).getSymbol()
+                        + "] [" + entries.get(intTaskIndex).getStatusIcon() + "] " + entries.get(intTaskIndex).description);
+                printLongLine();
+                break;
+            case "delete":
+                stringTaskIndex = userMessage.substring(userMessage.indexOf(" ") + 1);
+                intTaskIndex = Integer.parseInt(stringTaskIndex) - 1;
+                printLongLine();
+                System.out.println("Noted. I've removed this task:\n"
+                        + "   [" + entries.get(intTaskIndex).getSymbol()
+                        + "] [" + entries.get(intTaskIndex).getStatusIcon() + "] " + entries.get(intTaskIndex).description);
```

Comment:
Line is too long (more than 120 characters). Consider separating the line into multiple lines.